jupyterlab_comments/LICENSE

```diff
@@ -0,0 +1,28 @@
+BSD 3-Clause License
```

Comment:
Is there a specific reason not to use the Apache LICENSE from the root repository?